sdk/setup.py

```diff
@@ -1,17 +1,44 @@
-# coding: utf-8
+import setuptools
 
-"""
-    KFServing
+REQUIRES = [
+    'certifi >= 14.05.14',
+    'six >= 1.10',
+    'python_dateutil >= 2.5.3',
+    'setuptools >= 21.0.0',
+    'urllib3 >= 1.15.1',
+    'kubernetes >= 9.0.0',
+]
 
-    Python SDK for KFServing  # noqa: E501
-
-    OpenAPI spec version: v0.1
-    
-    Generated by: https://github.com/swagger-api/swagger-codegen.git
-"""
-
-
-from setuptools import setup, find_packages  # noqa: H301
-
-NAME = "kfserving"
-VERSION = "0.1"
+setuptools.setup(
+    name='kfserving',
```

Comment:
@ellis-bigelow what do you think about the name? I think we have the other package called kfserver ?

Reply:
Personally I set it to `kfserving` as want to keep consistent with out product name. :-) 
Waiting for @ellis-bigelow inputs.  Thanks.

Reply:
How about `kfserving-sdk`?

Reply:
Here's a thought. We already are planning on making our KFService and KFModel libraries available on pypi. Should these be in the same library? 

It is nontrivial overhead to manage the release of multiple packages like this. I don't imagine we will have so much code that it will be prohibitive to have it in the same library. We can always split in the future as well.

What do you think?

Reply:
kubeflow pipelines sdk is called kfp
Why not just kfs?

Reply:
Makes sense.

Reply:
From the meeting, we planed to bundling the server and client into same pypi, and discuss more in the #268